pkg/cli/common.go

```diff
@@ -260,3 +261,21 @@ func UseLayers() bool {
 	}
 	return false
 }
+
+// DefaultFormat returns the default image format
+func DefaultFormat() string {
+	format := os.Getenv("BUILDAH_FORMAT")
+	if format != "" {
+		return format
+	}
+	return "oci"
+}
+
+// DefaultIsolation returns the default image format
+func DefaultIsolation() string {
+	isolation := os.Getenv("BUILDAH_ISOLATION")
+	if isolation != "" {
+		return isolation
+	}
+	return ""
```

Comment:
According to `buildah-bud(1)` the default should be `"oci"`.

Reply:
It would still be, but I guess I can make it more clear.